install.sh

```diff
@@ -167,6 +167,98 @@ mkdir -p "$OMS_CACHE/runtime-script"
 mkdir -p "$OMS_CACHE/startup-script"
 mkdir -p "$OMS_CACHE/trash"
 
+# install lolcat with apt-get if not found
+if [ ! -f "$OMS/lib/bin/lolcat" ]
```

Comment:
I don't know what do you mean this `"$OMS/lib/bin/lolcat"`

Reply:
No error with my copilot I think...

Reply:
and whereis say:
```bash

$whereis lolcat
lolcat: /usr/games/lolcat /usr/share/man/man6/lolcat.6.gz

```
so I have fail... xD